embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/payload/BackgroundActivityMessageTest.kt

```diff
@@ -18,7 +18,7 @@ internal class BackgroundActivityMessageTest {
     private val appInfo = AppInfo("fake-app-id")
     private val deviceInfo = DeviceInfo("fake-manufacturer")
     private val breadcrumbs = Breadcrumbs(
-        customBreadcrumbs = listOf(CustomBreadcrumb("fake-breadcrumb", 1))
+        viewBreadcrumbs = listOf(ViewBreadcrumb("fake-view", 109234098234))
```

Comment:
any reason we want to replace this instead of just removing it? This is totally fine if the answer is just that it's easier for the merge lol

Reply:
I remove this in #608